src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -237,6 +237,12 @@ $pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
   &.pf-m-chip-container .pf-c-data-toolbar__item {
     margin-top: var(--pf-c-data-toolbar__content--m-chip-container__item--MarginTop);
   }
+
+  &.pf-m-toggle-group-container {
+    > :nth-last-of-type(2) {
```

Comment:
Can you comment the intent of this selector, that it’s removing the spacer from the last thing that isn’t the expandable content?

Reply:
+1 ^ 
these changes are fine if they are necessary. I get to break another rule of react to implement but that's sort of the theme with this component, so no big deal at this point.

Reply:
@nicolethoen I don't yet know that implementing this is necessary, but the selector is there if we need it. Might be a good idea see how the updates affect the React implementation before baking it in. If it's unnecessary, we can remove it later. @mcoker wdyt?